docs/en/manuals/google-play-instant.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Google Play Instant
+brief: This manual explains how to create Google Play Instant game with Defold.
+---
+
+# Google Play Instant
+
+Google Play Instant enables native games to launch on devices running Android 6.0 without being installed.
+
+
+![GooglePlayInstant](images/gpi/gpi-try-now.png)
+
+## Publishing process
+
+To be able to publish your game as Google Play Instant app you need to set your project properly:
+
+1. Create custom `AndroidManifest.xml` file and add next two attributes to the `<manifest>` element:
+
+```lua
+xmlns:dist="http://schemas.android.com/apk/distribution"
+android:targetSandboxVersion="2"
+```
+How it should looks with default AndroidManifest.xml for example:
+
+```lua
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="{{android.package}}"
+        android:versionCode="{{android.version_code}}"
+        android:versionName="{{project.version}}"
+        android:installLocation="auto"
+        xmlns:dist="http://schemas.android.com/apk/distribution"
+        android:targetSandboxVersion="2">
+```
+
+2. Add a dependency to the Google Instant Apps extension in your “game.project” file. Add “https://github.com/AGulev/instantapp/archive/master.zip” or point to the ZIP file of a [specific release](https://github.com/AGulev/instantapp/releases) to the Dependencies property.
```

Comment:
We should move this to defold/extension-instantapp at some point to make it more official.